diffrax/path.py

```diff
@@ -39,16 +39,26 @@ def evaluate(self, t0, t1=None, left=True):
     t0: Scalar = field(init=False, repr=False)
     t1: Scalar = field(init=False, repr=False)
 
+    def eval_levy(
+        self, t0: Scalar, t1: Optional[Scalar] = None, left: bool = True
+    ) -> LevyVal:
+        return self.evaluate(t0, t1, left, use_levy=True)
```

Comment:
I think we agreed this method shouldn't exist?

Reply:
Yes, we did, I apparently forgot to delete it. It is certainly not used anymore, I'm pretty sure.